Update identify-the-source-of-packages-with-digital-signatures.md

KrishnakumarRukmangathan opened the pull request:
Documentation bug:  2 issues on this article
1. The registry path provided for BlockedSignatureStates is incorrect. Changed this to correct registry path for SQL Server 2019.
2. Added a note to let the customers know that the registry path will be under 150 for SQL Server 2019, 140 for SQL Server 2017, 130 for SQL Server 2016, 120 for SQL Server 2014 and 110 for SQL Server 2012.

PRMerger20 commented:
@KrishnakumarRukmangathan : Thanks for your contribution! The author(s) have been notified to review your proposed change.

KrishnakumarRukmangathan commented:
Documentation bug:  2 issues on this article
1. The registry path provided for BlockedSignatureStates is incorrect. Changed this to correct registry path for SQL Server 2019.
2. Added a note to let the customers know that the registry path will be under 150 for SQL Server 2019, 140 for SQL Server 2017, 130 for SQL Server 2016, 120 for SQL Server 2014 and 110 for SQL Server 2012.

KrishnakumarRukmangathan commented:
Got closed by mistake. Reopening it

chugugrace commented:
#sign-off

ktoliver commented:
Hi @KrishnakumarRukmangathan - Thank you for your contribution. Would you take a moment to sign the Contributor License Agreement (CLA)? (Commits over a certain threshold trigger a CLA requirement.) After the CLA is signed, we can merge your pull request. Thanks!

ktoliver commented:
Email sent to internal contributor.

PRMerger18 commented:
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.
